lib/utils.js

```diff
@@ -247,13 +247,6 @@ function _asyncReadfile (filename) {
  * @return {string} A content type string
  */
 function getContentType (headers) {
-  const headerValue = headers.get ? headers.get('content-type') : headers['content-type']
-
-  // Default content type as stated by RFC 822
```

Comment:
RFC 822 doesn't even contain the string "text/plain" and predated HTTP, so completely bogus.

Reply:
Yes. For completeness' sake, [RFC 1521](https://tools.ietf.org/html/rfc1521) (since obsoleted) would have been the appropriate reference --

> `When a mail reader encounters mail with an unknown Content-type`
> `value, it should generally treat it as equivalent to`
> `"application/octet-stream", as described later in this document.`